sdk/identity/identity/test/manual-integration/AzureVM/index_user.ts

```diff
@@ -0,0 +1,21 @@
+import { SecretClient } from "@azure/keyvault-secrets";
+import { ManagedIdentityCredential } from "@azure/identity";
+
+async function main(): Promise<void> {
+  // DefaultAzureCredential expects the following three environment variables:
```

Comment:
Not using `DefaultAzureCredential` here; maybe we should? And is it possible to test system/user-assigned identities with the same code?

Reply:
I couldn't figure out a way to easily test both at once, so I separated them.  I'll remove the comment at least.